src/els_execute_command_provider.erl

```diff
@@ -46,6 +47,25 @@ execute_command(<<"replace-lines">>
             },
   els_server:send_request(Method, Params),
   [];
+execute_command(<<"info">>
+               , [#{ <<"uri">>   := _Uri }] = _Arguments) ->
+  {ok, Version} = application:get_key(?APP, vsn),
+  BinVersion = list_to_binary(Version),
+  Root = filename:basename(els_uri:path(els_config:get(root_uri))),
+  ConfigPath = case els_config:get(config_path) of
+                 undefined -> <<"undefined">>;
+                 Path -> list_to_binary(Path)
```

Comment:
😄 There's now an `els_utils:to_binary/1` function.

Reply:
Not in current master, that I can see.

https://github.com/erlang-ls/erlang_ls/blob/99e624c843a0fe50868a6c9614dd1dae08ce6bb3/src/els_utils.erl#L3-L12